.github/workflows/check-paper-integrity.yml

```diff
@@ -0,0 +1,20 @@
+name: Test Paper Architecture integrity
+on: [pull_request]
```

Comment:
I would suggest running this CI on PRs to main & when files inside `src/fabric` or `android/src/paper/java/.../viewmanagers` are edited.

Reply:
Fixed in https://github.com/software-mansion/react-native-screens/pull/2224/commits/8e4f7a0e7df8b0768a9fc33eecae16c4be3f7a50